tfaot-compile.file

```diff
@@ -17,7 +23,8 @@ cms_tfaot_compile \
   --aot-config "%{aot_config}" \
   --tool-name "%{n}" \
   --tool-base "%{i}" \
-  --output-directory compiled_model
+  --output-directory compiled_model \
+  --additional-flags="--target_triple %{build_arch}"
```

Comment:
Thanks @riga, looks like this worked ( atleast ppc64le build was successful)

Reply:
@rida, I guess `cms_tfaot_compile` tool is used by developers to work with new models ... right? Do we want developers to explicitly pass `--additional-flags` or may be you can set some default values in side the script ?

Reply:
Exactly. When run with `--dev` the tool mimics what the `tfaot-compile.file` does. The default arch for the `--target_triple` is `x86_64` so I guess this should apply in most cases. However, in dev mode we could check which arch is currently used and issue a warning, together with instructions on how to change the triple.

Reply:
@smuzaffar I just added a warning in https://github.com/riga/cms-tfaot/commit/a03ccef81c610e051d3d7d172d675fe6c1ff17e2. Was your plan to move the project to cms-externals? If not, I would adjust the commit hash in *this* PR.

Reply:
> Was your plan to move the project to cms-externals? If not, I would adjust the commit hash in this PR.

right, we should move it to cms-externals. I have added you developers of `cms-externals`. Can you please transfer the repo there. If you do not want to transfer then I can fork it

Reply:
I moved it and updated the reference in the spec.